puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_cli.yaml

```diff
@@ -35,10 +35,5 @@
           results: 'test/reports/*.xml'
           health-scale-factor: 1.0
       - ircbot_freenode
-      - trigger-parameterized-builds:
-        - project: release_nightly_build_rpm
-          condition: SUCCESS
-          predefined-parameters: |
-            project=packages/foreman/rubygem-hammer_cli
-            jenkins_job=${JOB_NAME}
-            jenkins_job_id=${BUILD_NUMBER}
+      - trigger-release-job:
```

Comment:
If you now trigger the release job, shouldn't that take care of the archiving? That'd mean you could remove the archive publisher (and need a matching `foreman-packaging` manifest change PR).

Reply:
Updated